services/crypto_create.proto

```diff
@@ -30,147 +37,184 @@ import "basic_types.proto";
+     * created accounts (i.e smart contracts) which default to -1.
+     * <p>
+     * If this value is `0`, then this account MUST manually associate to
+     * a token before holding or transacting in that token.<br/>
      * This value MAY also be `-1` to indicate no limit.<br/>
-     * This value MUST NOT be less than `-1`.<br/>
-     * By default this value is 0 for accounts except for auto-created accounts which default -1.
+     * This value MUST NOT be less than `-1`.
      */
     int32 max_automatic_token_associations = 14;
 
-    /**
-     * ID of the account or node to which this account is staking.
-     */
     oneof staked_id {
-
         /**
-         * ID of the account to which this account is staking.
+         * ID of the account to which this account is staking its balances.
+         * <p>
+         * If this account is not currently staking its balances, then this
+         * field, if set, MUST be the sentinel value of `0.0.0`.
          */
         AccountID staked_account_id = 15;
 
         /**
          * ID of the node this account is staked to.
+         * <p>
+         * If this account is not currently staking its balances, then this
+         * field, if set, SHALL be the sentinel value of `-1`.<br/>
+         * Wallet software SHOULD surface staking issues to users and provide a
+         * simple mechanism to update staking to a new node ID in the event the
+         * prior staked node ID ceases to be valid.
          */
         int64 staked_node_id = 16;
     }
 
     /**
-     * If true, the account declines receiving a staking reward. The default value is false.
+     * A boolean indicating that this account has chosen to decline rewards for
+     * staking its balances.
+     * <p>
+     * This account MAY still stake its balances, but SHALL NOT receive reward
+     * payments for doing so, if this value is set.
      */
     bool decline_reward = 17;
 
     /**
-     * The bytes to be used as the account's alias. It will be the
-     * serialization of a protobuf Key message for an ED25519/ECDSA_SECP256K1 primitive key type. Currently only primitive key bytes are
-     * supported as the key for an account with an alias. ThresholdKey, KeyList, ContractID, and
-     * delegatable_contract_id are not supported.
-     *
-     * May also be the EOA 20-byte address to create that is derived from the keccak-256 hash of a ECDSA_SECP256K1 primitive key.
-     *
-     * A given alias can map to at most one account on the network at a time. This uniqueness will be enforced
-     * relative to aliases currently on the network at alias assignment.
-     *
-     * If a transaction creates an account using an alias, any further crypto transfers to that alias will 
-     * simply be deposited in that account, without creating anything, and with no creation fee being charged.
+     * Bytes to be used as the account's alias.
+     * <p>
+     * This value, if set, MUST be one of the following values.<br/>
```

Comment:
Nit
```suggestion
     * This value, if set, MUST be one of the following values:<br/>
```

Reply:
I find the use of `:` a bit jarring, so I removed the `.` instead.